src/modules/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Typography } from '@mui/material';
```

Comment:
I think it is better to move it to common components and write some abstraction over it
Also don't forget to and storybook

Reply:
@VladSolodjankin Moved it to common please take a look and I'll add the stories soon in the next PR

Reply:
sorry looks like I confused you. We don't use  @mui/material directly in project, We write wrappers over it. You can look on Icon as an example.

Reply:
@VladSolodjankin Updated the PR